Packs/Expanse/Integrations/Expanse/Expanse.yml

```diff
@@ -655,7 +655,31 @@ script:
     - contextPath: Expanse.Exposures.Exposures.Provider
       description: Provider details of the Exposure
       type: Unknown
-  dockerimage: demisto/python3:3.8.2.6981
+  - arguments:
+    - default: true
+      description: The certificate common name.
+      isArray: true
```

Comment:
should this is `true`? you're not handling it in the code

Reply:
You're right, should be `false`.